examples/access_traits/main.cpp

```diff
@@ -58,23 +61,19 @@ int main(int argc, char *argv[])
   Kokkos::initialize(argc, argv);
 
   int const N = 10000;
-  float *a;
   float *d_a;
 
-  a = (float *)malloc(sizeof(float) * N);
-  cudaMalloc((void **)&d_a, sizeof(float) * N);
+  std::vector<float> a(N);
```

Comment:
Yes, that's better. Of course, you could also use `std::array` but it doesn't really matter for an example.